src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+$day-size: 100px;
+$gap-size: 1px;
+$columns: 7;
+$padding-size: 10px;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding-size;
+  gap: $gap-size;
+  max-width: calc(
+    #{$columns} * #{$day-size} + (#{$columns} - 1) * #{$gap-size} + 2 * #{$padding-size}
+  );
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to avoid using `calc` with SASS variables if the calculation can be done directly in SASS. Consider performing the arithmetic operation directly within SASS.